tensorrt_llm/_torch/pyexecutor/sampler/sampler.py

```
@@ -3811,10 +3784,10 @@ def sample_async(
         (
             requests,
             seq_slots_host,
+            seq_lens_host,
```

Ruff (0.15.20)

[warning] Unpacked variable `seq_lens_host` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)